vtStorManaged.ATest/PreBuild.py

```diff
@@ -0,0 +1,42 @@
+# Import built-in modules
```

Comment:
Since the managed code projects no longer reference unmanaged code libraries, we have to copy the unmanaged libraries to the output directory.